### ISSUE TITLE
Screen reader compatibility (accessibility) of control bar menus

### ISSUE THREAD

**Issue by OwenEdwards** [videojs/video.js#2769]
The fundamental structure of the 'menu buttons' using ARIA roles does not work because the div with `role="button"` which is the top-level element of the menu button contains actionable elements (menu items) as children. This is prohibited (although it's a little difficult to tease out of the ARIA spec exactly where that prohibition is spelled out). As a result, screen readers do not read out the menu structure correctly.

This is related to, but orthogonal to, #2155, and linked to #841, #1499, #1932 and #2136.

**Comment by gkatsev** [comment]
Fixed by #3033.
